coalib/parsing/FilterHelper.py

```diff
@@ -33,3 +33,22 @@ def get_filtered_bears(cls, filter, args, all_bears=None):
         if not args or len(args) == 0:
             return all_bears
         return cls.available_filters[filter](all_bears, args)
+
+    @classmethod
+    def filter_bears(cls, bears, args):
+        """
+        Returns bears after filtering based on ``args``. It returns
+        intersection of bears if more than one element is present in ``args``
+        list.
+
+        :param bears:   The bears to filter
```

Comment:
you can remove 2 spaces before the actual descriptions:
```
:param bears: The bears to filter.
# instead of
:param bears:   The bears to filter.
```

\+
Add periods at the end of every description ;)